package.json

```diff
@@ -7,6 +7,7 @@
     "axios": "^0.18.0",
     "express": "^4.16.3",
     "express-http-to-https": "^1.1.4",
+    "fecha": "^2.3.3",
```

Comment:
Where did you use this dependency

Reply:
Hi. I plan to use fecha to parse date on the Profile page. It's really lightweight and at 3kb gzipped it's a better alternative to Moment.js

Reply:
If you have a better/more efficient way of parsing dates, please let me know.

Reply:
Actually I saw this PR so i judged it that way it would be better if you introduced it just as you used it otherwise for this PR it looks like a unused dependency :slightly_smiling_face: 

You can keep it if you are going to use it

Reply:
I'm currently using it for the next pull.